documentation/source/user_section/command-line/sct_deepseg.rst

```diff
@@ -112,9 +115,9 @@ You can replace "``seg_sc_lesion_t2w_sci``" with any of the task names in the ta
    * - |seg_sc_lesion_t2w_sci| ``seg_sc_lesion_t2w_sci``
      - |seg_sc_ms_lesion_stir_psir| ``seg_sc_ms_lesion_stir_psir``
      - |seg_ms_lesion_mp2rage| ``seg_ms_lesion_mp2rage``
-   * - |seg_tumor_edema_cavity_t1_t2| ``seg_tumor_edema_cavity_t1_t2``
+   * - |seg_ms_lesion| ``seg_ms_lesion``
```

Comment:
Actually, now that I think about it, because your model is contrast agnostic, and thus probably our new "go-to" lesion segmentation model, shouldn't we put it first in the "Pathologies" table?

That way, we can list the model in the "Pathology segmentation can be performed by running the following sample command:" blurb, so that people are nudged to use your model first before trying the others.

Reply:
Yes I think it would make sense to at least put it before the other MS models. However, I don't know if it makes sense to put it before the SCI model. I guess it depends on if the SCI model is more used than MS models. @jcohenadad Which would you put first ?

Reply:
After discussion with @valosekj we decided to keep the sci model first and then have the 3 ms models consecutively.